public/src/main/java/com/revenuecat/purchases/LogLevel.kt

```diff
@@ -0,0 +1,28 @@
+package com.revenuecat.purchases
+
+enum class LogLevel {
+    VERBOSE,
+    DEBUG,
+    INFO,
+    WARN,
+    ERROR;
```

Comment:
Hmm I've just thought about it but it might have been good to add a way to disable logs altogether... Not that important nor a blocker for this though.

Reply:
That's fair, but I think if you want to do that you can set your own handler and then ignore them?